crates/fluvio/src/config/tls.rs

```diff
@@ -130,24 +129,16 @@ impl Debug for TlsCerts {
 }
```

Comment:
Just to be safe, bump fluvio up major version

Reply:
bumped to 0.23.0